disassemblers/adreno/__init__.py

```diff
@@ -10,7 +10,7 @@ def disasm(buf):
     shared = pathlib.Path(__file__).parent / "disasm.so"
     if not os.path.isfile(shared):
       os.system(f'cd {pathlib.Path(__file__).parent} && gcc -shared disasm-a3xx.c -o disasm.so')
-    fxn = ctypes.CDLL(shared)['disasm']
+    fxn = ctypes.CDLL(str(shared))['disasm']
```

Comment:
hmm, I think there's a method for this? curious why you need it

Reply:
I got an error from mypy saying that the parameter should be a string and not Path